django/db/models/query.py

```diff
@@ -2517,7 +2532,9 @@ def prefetch_one_level(instances, prefetcher, lookup, level):
         single,
         cache_name,
         is_descriptor,
-    ) = prefetcher.get_prefetch_queryset(instances, lookup.get_current_queryset(level))
+    ) = prefetcher.get_prefetch_querysets(
+        instances, lookup.get_current_querysets(level)
+    )
```

Comment:
During the deprecation period, this should fallback to `get_prefetch_queryset()` when `get_prefetch_querysets()` is not implemented.